code/modules/projectiles/projectile.dm

```diff
@@ -985,7 +997,7 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 
 	x_offset = round(sin(dir_angle), 0.01)
 	y_offset = round(cos(dir_angle), 0.01)
-	if(projectile_batch_move() == PROJECTILE_FROZEN)
+	if(projectile_batch_move(!recursivity) == PROJECTILE_FROZEN)
 		permutated.Cut()
```

Comment:
?

Reply:
it prevents the first part of the visual of the laser to appear (the one that comes directly from the gun). 

See where first projectile is used.